remo/featuredrep/views.py

```diff
@@ -11,11 +11,17 @@
 import forms
 from models import FeaturedRep
 
+from datetime import datetime
+
 
 @cache_control(private=True, max_age=60 * 10)
 def list_featured(request):
     """List all Featured Reps."""
-    featured_list = FeaturedRep.objects.all()
+
+    if not request.user.has_perms('featuredrep.can_edit_featured'):
+        featured_list = FeaturedRep.objects.filter(created_on__gte=datetime.today())
```

Comment:
adding the check here is not enough, that only works for the list itself (see my comment above for that part). However, users would still see future RoM on the frontpage:

![screen shot 2018-04-14 at 17 49 29](https://user-images.githubusercontent.com/330324/38769996-3c45e6e8-400c-11e8-97c8-89aa61ce9fdc.png)

Reply:
argh I missed that part!

Reply:
I can confirm that now is fixed show only the right profiles